### DISCUSSION TITLE
MkDocs

### DISCUSSION THREAD

**Discussion by Aviksaikat** [catppuccin/catppuccin#2391] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://www.mkdocs.org/dev-guide/themes/

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by ruslanlap** [comment]
Hi! 👋

You can try the **Catppuccin theme for MkDocs** here:
👉 [https://github.com/ruslanlap/mkdocs-catppuccin](https://github.com/ruslanlap/mkdocs-catppuccin)

I’ve also prepared detailed documentation with configuration examples and live demos:
👉 [https://ruslanlap.github.io/mkdocs-catppuccin/configuration/](https://ruslanlap.github.io/mkdocs-catppuccin/configuration/)

The theme supports different Catppuccin flavors and accent colors.
My personal favorite is **Macchiato + Green**, but you can easily change the accent color to any Catppuccin palette that fits your style.

If you have feedback, ideas, or would like to suggest improvements — feel free to share, I’d love to hear your thoughts! Thank you if you leave stars to project!
